snuba/subscriptions/scheduler.py

```diff
@@ -278,6 +283,43 @@ def add_tag(tags: Tags, builder_type: str) -> Tags:
         ]
 
 
+def filter_subscriptions(
+    subscriptions: List[Subscription],
+    entity_key: EntityKey,
+    metrics: MetricsBackend,
+    slice_id: Optional[int] = None,
+) -> List[Subscription]:
```

Comment:
```suggestion
) -> MutableSequence[Subscription]:
```

Reply:
FWIW, I think this type is copied from self.__subscriptions so it should probably be updated in both places or neither

Reply:
I can change it back!